tests/compile/function/test_pfunc.py

```diff
@@ -1,12 +1,14 @@
 import numpy as np
 import pytest
 
-import aesara
-from aesara import tensor as aet
-from aesara.compile.function import pfunc
+import aesara.tensor as aet
+from aesara.compile import UnusedInputError
+from aesara.compile.function import function, pfunc
 from aesara.compile.io import In
 from aesara.compile.sharedvalue import shared
 from aesara.configdefaults import config
+from aesara.configdefaults import config as aesara_config
```

Comment:
This import is unnecessary, because the same object is imported right in the line above.

Reply:
Done!